reliability/README.md

```diff
@@ -1,81 +1,205 @@
+| scaleup  | apps | admin/developer | 1/n | N/A | n% | scaleup n% of apps under the user |
+| scaledown  | apps | admin/developer | 1/n | N/A | n% | scaledown  n% of apps under the user |
+| login  | session | admin/developer | 1/n | N/A | N/A | login with the user |
+| clusteroperators  | monitor | N/A | N/A | N/A | N/A | check cluster operator by admin |
+| customized oc command  | customize | admin/developer | 1/n | N/A | N/A | run the customized oc command |
+| file with customized oc command  | customize | admin/developer | 1/n | N/A | N/A | run the file with customized oc command |
+
+
+```yaml
+reliability:
   tasks:
     minute:
+      # Specify an oc command to execute as 'action'.
+      # Don't use command that could return '1' as expected, e.g. oc get pods -A | egrep -v "Running|Completed".
+      # Use oc get pods -A | awk '$4!="Running" && $4!="Completed"' instead.
+      - action: oc whoami
+        resource: customize
+        persona: developer
+        concurrency: 5
+      # Specify a file to execute as 'action'.
+      # File contains lines of oc command to execute. Don't use command that could return '1' as expected.
+      # - action: <path to file>
+      #   resource: customize
+      #   persona: admin
+      #   concurrency: 1
       - action: check
         resource: pods
+        persona: admin
+        concurrency: 1
       - action: check
         resource: projects
-      - action: create
-        resource: projects
-        quantity: 1
+        persona: developer
+        concurrency: 5
     hour:
       - action: check
         resource: projects
-      - action: visit
-        resource: apps
+        persona: developer
+        concurrency: 5
       - action: create
         resource: projects
-        quantity: 3
+        quantity: 2
+        persona: developer
+        concurrency: 5
+      - action: visit
+        resource: apps
+        applyPercent: 100
+        persona: user
+        concurrency: 10
       - action: scaleUp
         resource: apps
         applyPercent: 50
+        persona: developer
+        concurrency: 3
       - action: scaleDown
         resource: apps
-        applyPercent: 100
+        applyPercent: 50
+        persona: developer
+        concurrency: 1
       - action: build
         resource: apps
-        applyPercent: 50
+        applyPercent: 33
+        persona: developer
+        concurrency: 2
+      - action: modify
+        resource: projects
+        applyPercent: 25
+        persona: developer
+        concurrency: 2
+      - action: clusteroperators
+        resource: monitor
     week:
       - action: delete
         resource: projects
-        applyPercent: 30
+        applyPercent: 33
+        persona: developer
+        concurrency: 5
       - action: login
         resource: session
-        user: kubeadmin
-        password: <password>
+        persona: developer
+        concurrency: 5
+
 ```
 
+## Git
```

Comment:
Might be good to add these sections (Git, Install dependencies, run and control activity execution) to below the files needed section /more toward the top so you don't have to read as much if you are just wanting to know how to install and run

Reply:
Good suggestion from reader's point of view!